### DISCUSSION TITLE
Improving ease of contribution for beginner student contributors

### DISCUSSION THREAD

**Discussion by itsmingjie** [Leaflet/Leaflet#7732] [General]
Dear Leaflet maintainers,

We're reaching out on behalf of an ongoing partnership between California State University Monterey Bay's [Computing Talent Initiative](https://computingtalentinitiative.org/) and [CodeDay](https://www.codeday.org/), a nonprofit organization. We're working on a program for college students who are new to contributing to open-source software, and we'd love to invite them to help closing some "good first issues" in your project! 

One of the goals of our program is to get more students to become comfortable with contributing to larger open source projects. A large codebase can be really intimidating to look at at first, so we'd love to invite you to host a short webcast to help students establish a mental modal of your codebase! 

If you are interested, our team (cc @SchmitzAndrew) will work with you to create/improve beginner-friendly documentation on how the code is structured, and how to get a local development environment running. We'll also provide primary mentorship to the students. In exchange, we'll need 30-45 minutes of time from a maintainer, and a commitment to be friendly to the students who are opening their first pull request. 

I'm open to discussing more below if you have questions! You can also get in touch with me directly at [mingjie@codeday.org](mailto:mingjie@codeday.org). 

Mingjie, Program @CodeDay
Berkeley CS'24

---
_If you would not like to participate, please feel free directly close this issue — sorry for the interruption!_

**Comment by Thewingpk1** [comment]
Hi Hope you are doing well
In which language do you want development.
Thanks 
Regards [TheWing.Pk](https://thewing.pk/)
